src/main/java/jobmanager/messaging/handler/RequestJobHandler.java

```diff
@@ -114,17 +79,21 @@ public void process(PiazzaJobRequest jobRequest, String jobId) {
 			}
 			// Commit the Job metadata to the Jobs table
 			accessor.addJob(job);
+
 			// Send the content of the actual Job under the
 			// topic name of the Job type for all workers to
 			// listen to.
-			ProducerRecord<String, String> message = JobMessageFactory.getWorkerJobCreateMessage(job, SPACE);
-			producer.send(message);
+			String queueName = String.format(JobMessageFactory.KAFKA_TOPIC_TEMPLATE, job.getJobType().getClass().getSimpleName(), SPACE);
```

Comment:
consider renaming this topic template to not contain the word kafka, since it's not specific to kafka.